tests/haproxy/node_rotate.yml

```diff
@@ -0,0 +1,48 @@
+# This file sets environment variables needed to run openstack commands on the bastion and then brings down the master nodes in turn, waiting until the last one comes up to bring down the next.
+- hosts: localhost
+  environment:
+    OS_AUTH_URL: "{{ osAuthUrl }}"
+    OS_TENANT_ID: "{{ osTenantId }}"
+    OS_TENANT_NAME: "{{ osTenantName }}"
+    OS_USERNAME: "{{ openstackOpenshiftUsername }}"
+    OS_PASSWORD: "{{ openstackOpenshiftPassword }}"
+  
+  tasks:
+  - name: reboot master-0
```

Comment:
again, this will stop working when we change the node names. You could pass in the node names from master-failover-test.sh, or you could work them out at the top of this playbook.

Reply:
another option is to use the fact that the master nodes are listed in [masters] in the inventory, and run this playbook on each one. You can't say "hosts: masters", because you want this to run ON THE BASTION, but you might be able to fish the list out of the inventory. (try hostvars[groups['masters'] as in this file: https://github.com/UKCloud/openshift-deployment-ansible/blob/feature/backup-tools/tools/playbooks/backup.yaml)